pkg/storage/calico/api_object_versioner.go

```diff
@@ -30,14 +26,5 @@ func (a APIObjectVersioner) ObjectResourceVersion(obj runtime.Object) (uint64, e
 	if len(version) == 0 {
 		return 0, nil
 	}
```

Comment:
We can remove this APIObjectVersioner completely now then.

That said, profiles probably need to use this but currently aren't - although I'm thinking it would be nice to remove profiles from the api server (except perhaps for etcd only).